### ISSUE TITLE
How to create and push to a release from different jobs?

### ISSUE THREAD

**Issue by develohpanda** [softprops/action-gh-release#165]
Hi! Thank you for this action 😄

I am trying to set up a workflow in which I have one job to create a release against a tag, and then a subsequent matrix which packages the app on three operating systems. From this matrix job, I would like to push assets to the release that was initially created.

Is it possible to use this action to achieve something like this? The two jobs & steps in question are highlighted below. Is there a recommended way to use the [outputs](https://github.com/softprops/action-gh-release#outputs) from this action when trying to upload artifacts?

<img width="738" alt="image" src="https://user-images.githubusercontent.com/4312346/135569740-c3191fff-94d6-4648-bdb5-459cd6cff8ff.png">

**Comment by softprops** [comment]
Yes the only want to refer to files created across jobs is with the upload/download artifact actions. 

Your use case should work with this action as long as the files exist in the locations you specify at the step you specify them for uploading to a release

**Comment by Hu1buerger** [comment]
> Yes the only way

**Comment by RadxaYuntian** [comment]
I think he is asking a use case similar to `actions/upload-release-asset`, a consumer for `outputs.upload_url`, since that action is deprecated and is pointing to this action instead.

**Comment by gutenye** [comment]
I end up using https://github.com/svenstaro/upload-release-action instead
